### ISSUE TITLE
[UI] Hozitontal scrolling in the app

### ISSUE THREAD

**Issue by DMtgd220** [jarnedemeulemeester/findroid#843]
Change horizontal scrolling to vertical scrolling in collections and seasons. This would make navigation more consistent with the same scrolling everywhere (eg: in libraries)

Thank again for you work on this app, guys!

**Comment by Jcuhfehl** [comment]
This wouldn't make it consistent with "everywhere" since the home screen also has horizontal scrolling

**Comment by DMtgd220** [comment]
Yes, but our collections with 10+ movies would become much more convenient to navigate.

This is the same for TV shows with 2+ seasons.

**Comment by DMtgd220** [comment]
If you can, yes please !

**Comment by shanikr24588** [comment]
Thanx
